src/main/java/aeneas/models/Level.java

```diff
@@ -135,5 +153,38 @@ public static Level loadLevel(File file) throws IOException {
     return bullpen.pieces;
   }
 
-  public abstract LevelWidgetView makeCorrespondingView();
+  public abstract LevelWidgetView makeCorrespondingView(Model model);
+
+  public abstract RadioButton getButton();
+
+  public abstract String getIconName();
+
+  public void start() { active = true; }
+  public void stop() { active = false; }
+
+  public boolean isActive() {
+    return active;
+  }
+
+  /**
+   * Get the text for counting down the user to finished.
+   * @return The text to be displayed to the user, eg "Time Remaining: 50"
+   */
+  public abstract String getCountdownText();
```

Comment:
It's better to use the level widget views. I saw your comment on my other PR, and all we have to do is make an editorPanel and a playerPanel. You still really want widgets for each level type. Having a common count down field is a hack